challenges/identifying-shared-costs.md

```diff
@@ -52,11 +52,15 @@ If you have a strong handle on these subjects, continue on to better understand
+
 ## Why allocate shared costs?
-A foundational principle of FinOps is: “everyone takes ownership for their cloud usage.” The true key to understanding total cost of ownership is built upon transparency and accuracy, but unallocated shared costs hinders both of these. Without appropriately splitting costs that are shared, engineers and product managers lack a complete picture of how much their products are really costing.
+Practitioners often struggle with which shared costs should and shoold not be split up and charged (or shown) back to cloud users. A foundational principle of FinOps is: “everyone takes ownership for their cloud usage,” which would argue that 100% of shared costs should be charged or shown back. However, a counter argument would be that cloud service users should only be "charged" with costs that they arguably have control over generating. Which is correct? One helpful framework for deciding when to charge back shared costs and when not to might be this: the two purposes of charging back ANY costs (cloud or not) are 1) to measure team performance and 2) to incentivize good behavior on the part of resource consumers. Because the process of charging back shared costs involves investment of time and effort, only those shared costs over which team members can exert at least partial control should be charged back. Here's a test for when and when not to charge back: "if I charge back this shared cost, will the resulting data cause any cloud consumers to change their consumption in a way that benefits the organization?" If there are no conceivable scenarios where they would, the shared cost should not be split and charged back. If there are meaninful scenarios involving material amounts of spend where they would, then the cost should be split and charged back. The key to understanding total cost of ownership is built upon transparency and accuracy, and unallocated shared costs over which cloud users have control hinder both of these. Without appropriately splitting such shared costs, engineers and product managers lack a complete picture of how much their products are really costing.
```

Comment:
I love this additional data, but I think we're getting more into the proscriptive "when to allocate shared costs" vs why shared costs should be allocated at all.  Do you think it makes sense to move this later on in the document?  I'm thinking maybe the section of "How to do This" and Step 1: Identify what costs are shared, because the subject you're circling around is that.  what costs are shared are driven by this "when to know the costs are shared".  Thoughts?